x/auctions/keeper/hooks.go

```diff
@@ -30,57 +34,47 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, _ int64)
 	lastBid := k.GetHighestBid(ctx)
 
 	// Distribute the awards from the last auction
-	if isValidBid(lastBid) {
```

Comment:
updated this check because it was converting the address two times and simplified